Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contest/ContestDetailsPage.tsx

```diff
@@ -47,16 +47,14 @@ const ContestDetailsPage = () => {
     const { actions: { setPageTitle } } = usePageTitles();
     const navigate = useNavigate();
 
-    const contestTitle = useMemo(
-        () => `${contestDetails?.name}`,
-        [ contestDetails?.name ],
-    );
-
     useEffect(
         () => {
-            setPageTitle(contestTitle);
+            if (contestDetails) {
+                setPageTitle(contestDetails?.name);
+            }
         },
-        [ contestTitle, setPageTitle ],
+        // eslint-disable-next-line
+        [ contestDetails ],
```

Comment:
Is `setPageTitle` cousing problems if in deps?

Reply:
It could possible do. On my opinion it is better in the dependency array to put only dependencies, that we are sure if they change, we want to call this useEffect, in other way it can be called when we do not want it, this will lead to bugs.

Reply:
Fixed